### PR TITLE
Fix '/apiconfig/alerts' route

### PR DESCRIPTION
**Pull request by Martouta**
Fixes the alerts route for all services
image
And now `test/integration/api/alerts_controller_test.rb` also passes.

### CODE REVIEW

**Comment by Martouta** [comment]
Btw @thomasmaas , are we currently keeping in mind the analytics permissions as you told me or they are being skipped?

File: app/controllers/api/alerts_controller.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ class Api::AlertsController < FrontendController
   before_action :find_service
 
   def index
+    activate_menu :dashboard
```
Comment (thomasmaas) [inline app/controllers/api/alerts_controller.rb]:
```suggestion
    activate_menu :applications, :alerts
```

**Comment by thomasmaas** [comment]
in the navigation we check: https://github.com/3scale/porta/blob/vertical-navigation/app/views/shared/provider/navigation/_vertical_nav.html.slim#L33

**Comment by thomasmaas** [comment]
fixed in
https://github.com/3scale/porta/pull/125/commits/729af310c6cd7570aafeed7a79057db0c9a82b8d

**Pull request closed without merging**